### PR TITLE
Amend the UI fxml and css files to create a more complete UI.

### PR DESCRIPTION
**Pull request by proto-aiken-13**
Our application needs to have a unique and functional AI from the original AB3 design.

Let's finish the AI for good! Updated DarkTheme and MainWindow.fxml to show a more complete UI.

### CODE REVIEW

**Comment by proto-aiken-13** [comment]
Add a comment tag to be displayed in the UI.

Closes #54

File: src/main/resources/view/DarkTheme.css [hunk 1]
```diff
@@ -77,38 +79,42 @@
...
 .list-cell {
     -fx-label-padding: 0 0 0 0;
     -fx-graphic-text-gap : 0;
     -fx-padding: 0 0 0 0;
+    -fx-border-color: black;
 }
 
 .list-cell:filled:even {
-    -fx-background-color: #3c3e3f;
+    -fx-background-color: white;
```
Comment (lsyurea) [inline src/main/resources/view/DarkTheme.css:106]: I like that you changed background color to fit the theme.

File: src/main/resources/view/MainWindow.fxml [hunk 2]
```diff
@@ -11,19 +11,25 @@
 <?import javafx.scene.layout.StackPane?>
 <?import javafx.scene.layout.VBox?>
 
+<?import javafx.scene.control.TitledPane?>
+<?import javafx.scene.control.Label?>
```
Comment (lsyurea) [inline src/main/resources/view/MainWindow.fxml:15]: I like the adding of more control in javafx to fit the theme of npc_track

File: src/main/resources/view/MainWindow.fxml [hunk 3]
```diff
@@ -11,19 +11,25 @@
...
   <scene>
     <Scene>
       <stylesheets>
         <URL value="@DarkTheme.css" />
         <URL value="@Extensions.css" />
       </stylesheets>
-
       <VBox>
+        <HBox VBox.vgrow="ALWAYS" style="-fx-background-color: #1194f0;">
+          <Label text="npc_track" styleClass="title-header" HBox.hgrow="ALWAYS"/>
```
Comment (lsyurea) [inline src/main/resources/view/MainWindow.fxml:31]: I like the customisation of header "npc_track" which is a vastly different frame from the previous iteration.

File: src/main/resources/view/MainWindow.fxml [hunk 4]
```diff
@@ -32,28 +38,26 @@
...
+            <Insets top="10" right="10" bottom="10" left="10" />
           </padding>
-        </StackPane>
-
+          <StackPane fx:id="personListPanelPlaceholder" VBox.vgrow="ALWAYS" style="-fx-background-color: white;"/>
+        </VBox>
         <StackPane VBox.vgrow="NEVER" fx:id="resultDisplayPlaceholder" styleClass="pane-with-border"
-                   minHeight="100" prefHeight="100" maxHeight="100">
+                   minHeight="100" prefHeight="100" maxHeight="100" style="-fx-background-color: #4d4e4f;
+                   -fx-border-color: #4d4e4f;">
```
Comment (lsyurea) [inline src/main/resources/view/MainWindow.fxml:49]: Great job of adhering to the checkstyles by reformatting the codebase to add 8 spacing for the next line!

File: src/main/resources/view/DarkTheme.css [hunk 5]
```diff
@@ -251,7 +262,7 @@
 }
 
 .button:default {
-    -fx-background-color: -fx-focus-color;
+    -fx-background-color: #989ea6;
```
Comment (lsyurea) [inline src/main/resources/view/DarkTheme.css:265]: Great job of making the color scheme consistent to our planned UI design!

File: src/main/java/seedu/address/ui/PersonCard.java [hunk 6]
```diff
@@ -59,5 +61,8 @@ public PersonCard(Person person, int displayedIndex) {
         person.getTags().stream()
                 .sorted(Comparator.comparing(tag -> tag.tagName))
                 .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+        person.getComments().stream()
```
Comment (lsyurea) [inline src/main/java/seedu/address/ui/PersonCard.java:64]: I like the modification to show the names of the comment.

**Review by lsyurea — APPROVED** [review]
Overall the UI design was well done! LGTM.

**Pull request merged**